.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  pull_request:
+  push:
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
```

Comment:
This could probably just be 7.4, we don't support anything below that.

Reply:
OK, pushed a commit for both of those files